spec/rails_helper.rb

```diff
@@ -4,6 +4,7 @@
 # Prevent database truncation (or other real-world consequences) if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'spec_helper'
+require 'rspec/rails'
```

Comment:
Require 'rspec/rails' here and also enable line 38 so RSpec will locate the test of `error_controller_spec.rb` as a test of a controller.